files/ansible/tasks/call_harness.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Get CAP_SYS_ADMIN
+  shell: cat /proc/sys/kernel/perf_event_paranoid > /root/cap_sys_admin
```

Comment:
This will leave files and multiple calls on the same machine will overwrite each other. Needs to be an Ansible variable.